main.py

```diff
@@ -0,0 +1,52 @@
+import csv
+
+
```

Comment:
I assume that you need to add a code for creating dict (JSON in the future) here:
```
def create_json( coord1: float, coord2: float) -> None:
    print(....)
```